### ISSUE TITLE
Configuring JPMML-SparkML for use in Jupyter notebooks

### ISSUE THREAD

**Issue by maximuslee1226** [jpmml/pyspark2pmml#33]
After following vruusmann's advice, I am still getting the error from Jupyter notebook. I ran mvn -Dmaven.test.skip=true clean install and also ran the pyspark command with packages, but I am still getting that error.

Here is my code:
from pyspark2pmml import PMMLBuilder
#import org.jpmml.sparkml.PMMLBuilder

pmmlBuilder = PMMLBuilder(spark, df, rfModel) \
.putOption(rf, "compact", True) \
.putOption(rf, "keep_predictionCol", False) \
.verify(df.sample(5)) \
.buildFile("DFML_pipeline.pmml")

Here is the Jupyter Error:

---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-22-2cec15ff7930> in <module>
      2 #import org.jpmml.sparkml.PMMLBuilder
      3 
----> 4 pmmlBuilder = PMMLBuilder(spark, df, rfModel) \
      5 .putOption(rf, "compact", True) \
      6 .putOption(rf, "keep_predictionCol", False) \

~/opt/anaconda3/lib/python3.6/site-packages/pyspark2pmml/__init__.py in __init__(self, sc, df, pipelineModel)
     12                 javaPmmlBuilderClass = sc._jvm.org.jpmml.sparkml.PMMLBuilder
     13                 if(not isinstance(javaPmmlBuilderClass, JavaClass)):
---> 14                         raise RuntimeError("JPMML-SparkML not found on classpath")
     15                 javaPmmlBuilder = javaPmmlBuilderClass(javaSchema, javaPipelineModel)
     16                 self.sc = sc

RuntimeError: JPMML-SparkML not found on classpath

> RuntimeError: JPMML-SparkML not found on classpath

Have you tried adding the [JPMML-SparkML](https://github.com/jpmml/jpmml-sparkml) library to your PySpark application classpath as detailed in the README file?

Works perfectly:
```
$ export SPARK_HOME=/opt/spark-3.0.0-preview2/
$ $SPARK_HOME/bin/pyspark --packages org.jpmml:jpmml-sparkml:1.6.0
```

And then copy&paste the "DecisionTreeIris" example code from the README file.

TLDR: RTFM

_Originally posted by @vruusmann in https://github.com/jpmml/pyspark2pmml/issues/29#issuecomment-616032268_

**Comment by vruusmann** [comment]
> I am still getting the error from Jupyter notebook.

Then you need to fix your Jupyter notebook's classpath. I can't advise you here, because it's not my specialty.

Anyway, the "standard procedure" as outlined in the README file works perfectly. I've attached a `DecisionTreeIris.pmml` file that I just produced this way.

[DecisionTreeIris.pmml.txt](https://github.com/jpmml/pyspark2pmml/files/5877932/DecisionTreeIris.pmml.txt)

**Comment by maximuslee1226** [comment]
Thanks, I was able to get this solved and the trick was to do two operations together

1. Update the `~/.bash_profile` file with the following, and execute `source ~/.bash_profile`
```
export PYSPARK_DRIVER_PYTHON=jupyter
export PYSPARK_DRIVER_PYTHON_OPTS='notebook'
```
2. Then execute `pyspark --jars jpmml-sparkml-executable-1.5.11.jar` which will bring up Jupyter notebook

**Comment by vruusmann** [comment]
@maximuslee1226 Thanks - this is a valuable piece of information.

I've tweaked your comment a bit to make the file names and commands stand out better. I'll also tweak the title of this issue to make it findable by Google.
